wire-gradle-plugin/src/test/projects/sourcepath-no-sources/src/main/proto/dummy.md

```diff
@@ -0,0 +1,3 @@
+Added to keep this folder structure for a test
+
+https://stackoverflow.com/questions/115983/how-can-i-add-an-empty-directory-to-a-git-repository/21422128#21422128
```

Comment:
tricky

Reply:
an empty `.gitignore` also works

Reply:
was going to go that route, but .gitignore wouldn't appear in a naive ls, so chose the dummy route.